c++/src/capnp/rpc.c++

```diff
@@ -2815,6 +2841,23 @@ private:
     }
 
     RemotePromise<AnyPointer> send() override {
+      if (callBuilder.getIsRealtime()) {
+        // For proxied realtime calls, we need to use the streaming path to avoid
```

Comment:
This seems like an awkward place to handle proxying. Could we instead handle this in RpcCallContext::directTailCall(), making it use `sendStreaming()` if the call has the realtime hint?

Reply:
It's actually much clearer in directTailCall()! Thanks for the suggestion! 🚀